### PR TITLE
Add basic modeling tool

### LINKED ISSUE #3: Build basic modelling tool
As an RPA developer, I'd like to able to visit a modeling site where I can create BPMN models, in order to document my customer's process. 

---

### :thought_balloon: Hints & Thoughts
Check out https://bpmn.io/
- Simple install, should only be able to use it.
Do #2 first or in parallel
### :question: Additional Questions to Answer / Consider

---

## :white_check_mark: Conditions of satisfaction

- [x] I can move BPMN activities, events, gateways into the drawing area
- [x] I can connect the BPMN artifacts
- [x] I can move the artifacts in the drawing area

### LINKED ISSUE #4: Create Attribute Sideboard
As an RPA Developer, I'd like to be able to edit the name of single activities in the drawing area on a sideboard.

---

### :thought_balloon: Hints & Thoughts
Checkout our Mockup
Do #3 first
### :question: Additional Questions to Answer / Consider Maximilian's mesage:

> Ich hab selbst schon mal ne ganze Weile mit bpmn.io gearbeitet und ihn für einen Use-Case angepasst. Die Community ist da sehr aktiv und es gibt auch einige [offizielle Beispiel-Repos](https://github.com/bpmn-io/bpmn-js-examples/), die ziemlich viele Fälle abdecken. Unter anderem auch, wie man das [Attribut-Panel](https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel) einbindet, dieses Panel um eigene Elemente [erweitert](https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel-extension), oder sogar [eigene Elemente zum Modellieren](https://github.com/bpmn-io/bpmn-js-examples/tree/master/custom-elements) hinzufügt. 

---

## :white_check_mark: Conditions of satisfaction

- [ ] When clicking on an activity a sideboard appears and a field name can be edited
- [ ] When changing the name in the field the activity in the BPMN diagram changes

### PR DESCRIPTION
**Pull request by WolfgangDaniel**
Fixes #3 
Fixes #4 

## Changes
A bpmn-js Modeler is implemented into the project now. All the necessary requirements have been documented in the readme file.

## Screenshots
Before: nothing
After:
grafik

## Checklist before merge

**Developer's responsibilities**
* [ ] Approved by at least one developer: *enter reviewer here*
* [ ] **Pull request build** has passed
* [ ] Production backend has been updated to reflect new features and fix bugs
* [ ] tested locally (in all browsers if frontend)
* [ ] updated the docs
* [ ] added tests where necessary

**Product Owner's responsibilities**
* [ ] No issues have been found on staging environment
* [x] Product Owner has approved the release

### CODE REVIEW

**Review by lukashueller — APPROVED** [review]
Looks very good. The README.md is very helpful!

File: client/src/pages/Modeler.js [hunk 1]
```diff
@@ -1,21 +1,21 @@
-import React from 'react';
-import BtnGotoHome from '../components/BtnGotoHome';
-import {Typography} from 'antd';
+import React from "react";
+import BtnGotoHome from "../components/BtnGotoHome";
+import { Typography } from "antd";
 import BpmnModelerComponent from "../components/bpmn.modeler.component";
+import "bpmn-js-properties-panel/dist/assets/bpmn-js-properties-panel.css";
```
Comment (SanJSp) [inline client/src/pages/Modeler.js]: Single quotes

File: client/src/pages/Modeler.js [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-            <React.StrictMode>
-                <BpmnModelerComponent/>
-            </React.StrictMode>
-            {/*<BtnGotoHome/>*/}
-        </div>
-    );
-}
+  return (
+    <div>
+      {/*<Title>Seite unseres Modelers</Title>*/}
```
Comment (SanJSp) [inline client/src/pages/Modeler.js:12]: has this worked uncommented?

File: client/src/pages/Modeler.js [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+      <React.StrictMode>
+        <BpmnModelerComponent />
+      </React.StrictMode>
+      {/*<BtnGotoHome/>*/}
```
Comment (SanJSp) [inline client/src/pages/Modeler.js:16]: see above

**Review by SanJSp — CHANGES_REQUESTED** [review]

File: client/src/pages/Modeler.js (hunk 2 above)
Comment (WolfgangDaniel) [reply client/src/pages/Modeler.js:12]: yes

File: client/src/pages/Modeler.js (hunk 3 above)
Comment (WolfgangDaniel) [reply client/src/pages/Modeler.js:16]: yes

**Review by kej-jay — APPROVED** [review]

**Pull request merged**